generators/python/src/fern_python/generators/sdk/client_generator/oauth_token_provider_generator.py

```diff
@@ -549,18 +549,21 @@ def _get_refresh_function_invocation(
 
     def _get_expires_at_function_declaration(self) -> AST.FunctionDeclaration:
         """
-        def _get_expires_at(expires_in_seconds: int, buffer_in_minutes: int):
+        def _get_expires_at(expires_in_seconds: typing.Optional[int], buffer_in_minutes: int):
+            # Default to 1 hour if expires_in_seconds is not provided
+            DEFAULT_EXPIRY_SECONDS = 3600
+            effective_expires_in = expires_in_seconds if expires_in_seconds is not None else DEFAULT_EXPIRY_SECONDS
```

Comment:
Let's make it simpler since it's well documented
```suggestion
            effective_expires_in = expires_in_seconds if expires_in_seconds is not None else 3600
```